packages/training/deploy/docker/Dockerfile.base

```
@@ -0,0 +1,72 @@
+
+# Install system dependencies
+USER root
+RUN apt-get update && apt-get install -y \
+    curl \
+    wget \
+    unzip \
+    postgresql-client \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Bun for database migrations
+RUN curl -fsSL https://bun.sh/install | bash
```

Hadolint (2.14.0)

[warning] Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check

(DL4006)